### ISSUE TITLE
Ability to switch to pattern-differentiation from color-differentiation in graphs

### ISSUE THREAD

**Issue by quaid** [oss-aspen/8Knot#84]
**Please describe the background and context for this new visualization**

Using colors to represent differentiation is a known accessibility issue, for example, people have color differentiation issues, perceive colors differently than others, cannot see the same range of colors, are visually overwhelmed by too many colors, and so forth.

**Describe the perspective you'd like the final visual to give**

My suggestion is to have a black/white pattern that corresponds to every color you use in a chart (which could be available to see as a single legend on a pop-up out of the tile).

I reckon that a limited-palette grayscale might be OK, perhaps there is some clear science that lets us know that 256 colors would be too many but e.g. 30 shades of gray with black and white for the patterns.

**Acceptance criteria for the issue and visualization to be complete**
- [ ] Augur Query
- [ ] Preprocess data into pandas data frame format for visualization
- [ x ] Plotly graph creation
- [ ] Notes with any background context that would be useful for Explorer integration 
- [ ] Open issue on explorer for visualization integration using template

**Additional context**
I have a feeling you might be able to find an existing "palette" of colors and patterns that are known to be useful to people with all sorts of visual differences.
